field.cpp

```diff
@@ -709,6 +708,11 @@ int32 field::get_spsummonable_count_fromex_rule4(card* pcard, uint8 playerid, ui
 		if(!is_location_useable(playerid, LOCATION_MZONE, 6))
 			flag |= 1u << 6;
 	}
+	uint32 linked_zone = 0;
+	if(core.duel_rule >= 5 && pcard && (pcard->get_type() & TYPE_FUSION | TYPE_SYNCHRO | TYPE_XYZ) && pcard->is_position(POS_FACEDOWN))
```

Comment:
```suggestion
	if(core.duel_rule >= 5 && pcard && (pcard->data.type & TYPE_FUSION | TYPE_SYNCHRO | TYPE_XYZ) && !((pcard->data.type & TYPE_PENDULUM) && pcard->is_position(POS_FACEUP)))
```
compatible to
https://github.com/Fluorohydride/ygopro-core/blob/c87a077663dd3944e15269f9015708af00c343ef/card.cpp#L3099

Reply:
> エクストラデッキの**裏側表示**の融合・シンクロ・エクシーズモンスターは
自分のメインモンスターゾーンにも特殊召喚できる
https://www.yugioh-card.com/japan/howto/masterrule2020/